docs/VLSI/Tutorial.rst

```diff
@@ -54,7 +54,7 @@ Prerequisites
 * Genus, Innovus, and Calibre licenses
 * For ASAP7 specifically:
 
-  * Download the `ASAP7 PDK v1p5 <http://asap.asu.edu/asap/>`__ tarball to a directory of choice but do not extract it. The tech plugin is configured to extract the PDK into a cache directory for you.
+  * Download the `ASAP7 PDK v1p5 <http://asap.asu.edu/asap/>`__ tarball to a directory of choice but do not extract it. The tech plugin is configured to extract the PDK into a cache directory for you. Note: v1p5 of the PDK is not publicly available, and you will need to contact the developers for it. The v1p7 version that is `publicly released <https://github.com/The-OpenROAD-Project/asap7>`__ currently has too many bugs to be integrated into our flow.
```

Comment:
Could this be written a bit more diplomatically? (the "has too many bugs to be integrated into our flow" part).
For example, "has some critical issues involving X, Y, Z which currently prevent it from be integrated into the Hammer flow"

Reply:
I don't want to enumerate what's already in the issue list...